apps/server/src/services/PlaybackService.js

```diff
@@ -127,6 +127,29 @@ export class PlaybackService {
     }
   }
 
+  /**
+   * Starts playback on next event
+   * @param {string} [fallbackAction] - 'stop', 'pause', or null
+   */
+  static startNext(fallbackAction) {
```

Comment:
remember also to type typescript
`static startNext(fallbackAction?: 'stop' | 'pause' ) {`

Reply:
the file isn't a typescript file yet. Thats the reason why i left it out. do you want me ton convert it?

Reply:
Yes please. Let me know if you need help